Directory.Build.props

```diff
@@ -0,0 +1,78 @@
+<Project>
```

Comment:
There are formatting issues in this file.

Reply:
Will be fixed